modules/ROOT/nav.adoc

```diff
@@ -85,7 +85,9 @@
  ** xref:pce-configuration-macos.adoc[MacOS]
  ** xref:pce-configuration-windows.adoc[Windows]
 
-* xref:set-credentials-in-studio-to.adoc[Configure Platform Credentials]
+* xref:login-platform-from-studio.adoc[Login to Anypoint Platform]
+ ** xref:set-credentials-in-studio-to.adoc[Configure Platform Credentials]
```

Comment:
```suggestion
* xref:login-platform-from-studio.adoc[]
 ** xref:set-credentials-in-studio-to.adoc[]
```
You can remove these if the article's titles are correct. If you don't remove them, please change **Login** with **Log in** as suggested in the next comment.

Reply:
Let’s change the title and make it right ;) 
Thanks!